templates/Collector/db.html.twig

```diff
@@ -468,6 +480,34 @@
             return false;
         }
 
+        function run(link) {
+            "use strict";
+
+            var targetId = link.getAttribute('data-target-id');
+            var targetElement = document.getElementById(targetId);
+
+            if (targetElement.style.display != 'block') {
+                if (targetElement.getAttribute('data-sfurl') !== link.href) {
+                    fetch(link.href, {
+                        headers: {'X-Requested-With': 'XMLHttpRequest'}
+                    }).then(async function (response) {
+                        targetElement.innerHTML = await response.text()
+                        targetElement.setAttribute('data-sfurl', link.href)
+                    }, function () {
+                        targetElement.innerHTML = 'An error occurred while loading the query explanation.';
+                    })
+                }
+
+                targetElement.style.display = 'block';
+                link.innerHTML = 'Hide query result';
+            } else {
+                targetElement.style.display = 'none';
+                link.innerHTML = 'Run query';
+            }
+
+            return false;
+        }
```

Comment:
Please parametrize the function that was copied instead of copying whole thing and just changing few lines